spec/wikidata/diff/api_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# spec/wikidata/diff/api_spec.rb
+
+require './lib/wikidata/diff/api'
+require 'rspec'
+
+RSpec.describe Api do
+    describe '.get_revision_contents' do
+      let(:revision_ids) do
+        [
+          2266123021, 2266341034, 2266123060, 2266123123, 2266123148,
+          2266123175, 2266123210, 2266123270, 2266123325, 2266123373,
+          2266123418, 2266341148, 2266123442, 2266123459, 2266123479,
+          2266123502, 2266123529, 2266123536, 2266123548, 2266123562,
+          2266123568, 2266341782, 2266123581, 2266123596, 2266123602
+        ]
+      end
+
+      it 'returns a truncation warning since query exceeds response size limit' do
```

Comment:
This test isn't actually running any of the gem code, it's replicating the client setup and just testing against the mediawiki_client gem. It should probably be rewritten to use `.mediawiki_request` method, which at least will involve a little bit of gem code and might break if the implementation is changed in significant way (and could be extended with more tests if we discover and add fixes for additional bugs).